### DISCUSSION TITLE
Support for transformer models

### DISCUSSION THREAD

**Discussion by hanshupe007** [MAIF/shapash#466] [General]
I would like to use shapash to analyse fine-tuned BERT models from huggingface. Is this currently supported?

**Comment by ThomasBouche** [comment]
Hi,
Shapash does not support this type of model. 
Shapash is not currently designed for NLP models. Perhaps a library like shap can be used for your model.

Shapash works with models: Catboost, Xgboost, LightGBM, Sklearn Ensemble, Linear models, SVM
